l10n_es_facturae/data/data_res_country.xml

```diff
@@ -0,0 +1,742 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Yo tb invertí mucho tiempo desde la versión 5 y no digo nada de eso, para
esto existe comunidad, en este pr mi aportación es la migración de este
módulo a la 8 si tu no quieres tener tantas lineas en la localización
propón uno pr con una refactorización y yo te lo aprobare y mergeare, yo no
tengo porque hacerlo, en mi pr ese no era mi objetivo.

In l10n_es_facturae/data/data_res_country.xml
<https://github.com/OCA/l10n-spain/pull/214#discussion_r39677834>:

> @@ -0,0 +1,742 @@
> +<?xml version="1.0" encoding="utf-8"?>

Por eso precisamente se quedan como PRs, porque prefiero que no pasen esas
cosas. Creo que no es un cambio tan grande el que te pido, y tú sabes que
es lógico que vaya de la forma que te comento. Si no "puedes invertir" más
tiempo en ello, piensa en el tiempo que invierto yo en que tengas muchas
cosas ya hechas, incluidas muchas mejoras en la localización, así que
tampoco pasa nada por ser un poco solidario con ello. Mi tiempo para
revisarlo lo vas a tener.

—
Reply to this email directly or view it on GitHub
<https://github.com/OCA/l10n-spain/pull/214/files#r39677834>.